### DISCUSSION TITLE
The workflow is not valid. Unexpected value 'POSTGRES_PASSWORD'

### DISCUSSION THREAD

**Discussion by atitov-align** [community/community#25292] [Actions]
<p>Getting “The workflow is not valid. Unexpected value ‘POSTGRES_PASSWORD’”<br>
for below workflow:</p>
<pre><code class="lang-yaml"># .github/workflows/rspec.yml
  name: CI RSpec Tests
  
  on: [push, pull_request]
  
  jobs:
    build:
      name: CI
      runs-on: ubuntu-latest
      container: ruby:latest
  
      services:
        postgres:
          image: postgres:latest
          ports: ["5432:5432"]
          options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
        env:
          POSTGRES_PASSWORD: 'postgres' # this line fails
...
</code></pre>

**Comment by jsoref** [comment, marked as answer]
<p>You probably want to indent the env block and child one more level. It’s a property of the postgresql service, not its own service.</p>

**Comment by atitov-align** [comment]
<p>Thank you!</p>
<p>Bad insomnia consequences :man_shrugging:</p>
